lib/modem_info/modem_info.c

```diff
@@ -503,7 +507,15 @@ static void modem_info_rsrp_subscribe_handler(void *context, const char *respons
 		return;
 	}
 
-	err = modem_info_parse(modem_data[MODEM_INFO_RSRP],
+	const struct modem_info_data rsrp_data = {
+		.cmd			= AT_CMD_CESQ,
```

Comment:
Please fix the alignment in this structure.

Reply:
Sorry. Not clear about the alignment issue.

Reply:
![image](https://user-images.githubusercontent.com/22167799/79582963-c5e4a080-80cc-11ea-9bdc-23f44264d0f3.png)

`AT_CMD_CESQ`, `RSRP_DATA_NAME`, `AT_PARAM_TYPE_NUM_SHORT` are one tab off. Note, that we use the same formatting as Zephyr here, 8 spaces per tab.

Reply:
Thank you. I had my tabs set at 4. Fixed!